uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -270,10 +315,12 @@ async def asgi_receive(self):
         try:
             await self.ensure_open()
             data = await self.recv()
-        except websockets.ConnectionClosed as exc:
+        except ConnectionClosed as exc:
             return {"type": "websocket.disconnect", "code": exc.code}
 
-        msg = {"type": "websocket.receive"}
+        msg: WebSocketReceiveEvent = {  # type: ignore[typeddict-item]
+            "type": "websocket.receive"
+        }
```

Comment:
@graingert This is the statement I was talking about. I don't want to have the other keys on this dict, tho.